tools/cli/pkg/ers/command/logs.go

```diff
@@ -0,0 +1,129 @@
+package command
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"regexp"
+
+	"github.com/kyma-project/control-plane/tools/cli/pkg/logger"
+	"github.com/pkg/errors"
```

Comment:
replace `"github.com/pkg/errors"` with
`"errors"` as this is now part auf standard library and `pkg/errors` was archived